index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile first Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header class="main-header">
+
+            <nav class="header">
+
+                <div class="name"><a href="#" style="color: white; text-decoration: none;">Usama Masood</a></div>
+                <div class="menu"><i class="fas fa-bars"></i></div>
+                
+            </nav>
+    </header>
+
+    <main class="main1">
+          <section class="section1">
+            <div class="intro">
+                <h1>Hey There. <br>
+                    I'm Usama
+                </h1>
+            </div>
+
+            <div class="profession"><h2>I'am a Software Developer</h2></div>
+
+            <div class="container">
+                <div class="container1">
+                    <div>
+                        <article class="services">
+                            <p>
+                                I can help you build a product , feature or <br>
+                                 website Look through some of my work <br>
+                                  and experience! If you like what you see <br>
+                                 and have a project you need coded, <br>
+                                don’t hestiate to contact me. <br>
+                            </p>
+                        </article>
+                        <ul class="logo">
+                            <li><a href="https://github.com/UsamaMasood12"><img src="./images/Vector.png" alt=""></a></li>
+                            <li><a href="www.linkedin.com/in/usama-masood-b4a35014b"><img src="./images/Group.png" alt=""></a></li>
+                            <li><a href="#"><img src="./images/1.png" alt=""></a></li>
+                            <li><a href="https://twitter.com/Usama____Masood"><img src="./images/3.png" alt=""></a></li>
+                            <li><a href="#"><img src="./images/2.png" alt=""></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="container2">
+                    <div class="box box1"><img src="./images/4.png" alt=""></div>
+                </div>
```

Comment:
- [optional] Kindly please remove this image from here and use CSS background-image property to do that. * If an image is not part of the content it is always better to use CSS background-image property. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info. 👍